src/aggregator/integration/custom_aggregations_test.go

```diff
@@ -90,9 +90,7 @@ func testCustomAggregations(t *testing.T, metadataFns [4]metadataFn) {
 	instance := cfg.newPlacementInstance()
 	placement := newPlacement(numShards, []placement.Instance{instance})
 	placementKey := serverOpts.PlacementKVKey()
-	placementStore := serverOpts.KVStore()
-	require.NoError(t, setPlacement(placementKey, placementStore, placement))
-	serverOpts = serverOpts.SetPlacement(placement)
+	require.NoError(t, setPlacement(placementKey, serverOpts.ClusterClient(), placement))
```

Comment:
Nit: maybe you want to push the `require.NoError(t` check into the `setPlacement` for a bit of additional cleaning up? Besides, that would be consistent with `serverOpts = setupTopic(t, serverOpts, placement)` a line below.

Reply:
https://github.com/m3db/m3/pull/3745/commits/0833d738fc64b6709b5390cc67311ecec8dabdc2